### ISSUE TITLE
"expensive call check" with latest method

### ISSUE THREAD

**Issue by dreinull** [nunomaduro/larastan#548]
- Larastan Version: 0.5.6
- `--level` used: 1

### Description

Looks like the "expensive call" check has a false positive with the `latest()` method.

### Laravel code where the issue was found

```php
public function scopeOfType($query, $name = 'default')
{
        return $query->where('type', $name)->latest();
}

Entity::ofType('something')->first();
```

Ends with the message: `Called 'first' on Laravel collection, but could have been retrieved as a query.`

If I replace `latest()` with `orderBy('created_at', 'DESC')`, which is the content of the latest method, the result is `[OK] No errors`

**Comment by nunomaduro** [comment]
@Daanra can you help on this one?

**Comment by Daanra** [comment]
Pretty sure this was fixed in https://github.com/nunomaduro/larastan/pull/546

I'll investigate it later today.

**Comment by Daanra** [comment]
Was indeed fixed by #546, and hence in [v0.5.7](https://github.com/nunomaduro/larastan/releases/tag/v0.5.7)

**Comment by canvural** [comment]
@DanielRuf thanks for investigating!

@dreinull Please try the 0.5.7, it should be fixed now :+1:
